api/src/Entity/ContentNode.php

```diff
@@ -51,6 +51,9 @@
 #[ORM\InheritanceType('SINGLE_TABLE')]
 #[ORM\DiscriminatorColumn(name: 'strategy', type: 'string')]
 #[ORM\UniqueConstraint(name: 'contentnode_parentid_slot_position_unique', columns: ['parentid', 'slot', 'position'])]
+#[ORM\Index(columns: ['createTime'])]
```

Comment:
Are the createTime and updateTime inices also new? Why are they not in the migration?

Reply:
Actually, these indexes are already on `BaseEntity`. If they are not repeated here, they are omitted. 
I suspect it is a bug in Doctrine.

Reply:
what do you mean by omitted?
they are in the schema and in the migrations. why do you need to add them again on the subclass entity?

Reply:
If I add only the `Index(category)` to `ContentNode`, then the two indexes of BaseEntity are lost. 
It looks as if the index attributes are not aggregated - but the mostly specific class with index attributes determines the list of indexes.